.changeset/forty-humans-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/qwik': major
+---
+
+crash down when there are two slashes at once
```

Comment:
Ok I think you can change this to something like "gracefully handle image dimensions service errors"